### ISSUE TITLE
Fix npm breakage with v3.0.0 release

### ISSUE THREAD

**Issue by jacobq** [mawie81/electron-window-state#8]
For some reason, `index.js` is not getting included in the bundle any more, so `npm install --save electron-window-state` creates a folder in `node_modules` with just the readme, license, and package.json file.
However, if one changes their package.json file (for project using electron-window-state) from this (also breaks with `*`, `3.x`, `~3.0.0`, or `3.0.0` for version though strangely works for `2.1.0` despite not being modified since then aside from the version field):
```
  "dependencies": {
    "electron-window-state": "^3.0.0"
  }
```
to this (or references commit SHA)
```
  "dependencies": {
    "electron-window-state": "git@github.com:mawie81/electron-window-state.git"
  }
```
and runs `npm install` again then it appears.

**Comment by jacobq** [comment]
BTW, this was discovered on a Linux 4.5.0-1-amd64 system with npm version 3.8.6 and node v5.11.1

**Comment by mawie81** [comment]
Thanks for the feedback.
Still don´t know why that happened. Had to republish as 3.0.2.

**Comment by mawie81** [comment]
For the record: this might have happened because I published using Node 6.
See https://github.com/npm/npm/issues/5082

**Comment by jacobq** [comment]
This has to be one of the weirdest things I've seen npm do. I spent about 15 minutes trying to figure it out before filing this so am at least relieved to know that I am not crazy. I saw that you had specified a `files: ['index.js']` property in `package.json` which should have been fine. I don't think deleting that and/or adding `main: 'index.js'` makes any difference -- just got lucky with race condition. Thankfully it sounds like this bug is related to publishing only so at least it isn't going to randomly break when people run `npm install`.
